spot-contracts/contracts/_interfaces/IBondIssuer.sol

```diff
@@ -8,10 +8,17 @@ interface IBondIssuer {
     /// @param bond The newly issued bond.
     event BondIssued(IBondController bond);
 
+    /// @notice Event emitted when a bond has matured.
+    /// @param bond The mature bond.
+    event BondMature(IBondController bond);
```

Comment:
```suggestion
    event BondMatured(IBondController bond);
```

Reply:
I don't want to change the existing interface which the other issuers on mainnet implement